templates/moinmoin/files/.platform.app.yaml

```diff
@@ -0,0 +1,36 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: app
+
+# The runtime the application uses.
+type: "python:2.7"
```

Comment:
Is it not Python 3 compatible?  What crummy software. :smile:

Reply:
Real pain is what it is.